container-chains/templates/frontier/runtime/src/migrations.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+//! # Migrations
+//!
+//! This module acts as a registry where each migration is defined. Each migration should implement
+//! the "Migration" trait declared in the pallet-migrations crate.
+
+use {
+    crate::{Precompiles, LOG_TARGET},
+    frame_support::weights::Weight,
+    pallet_migrations::{GetMigrations, Migration},
+    sp_core::Get,
+    sp_std::{marker::PhantomData, prelude::*},
+};
+
+pub struct MigratePrecompileDummyCode<T>(pub PhantomData<T>);
+impl<T> Migration for MigratePrecompileDummyCode<T>
+where
+    T: pallet_evm::Config,
+    T: frame_system::Config,
+{
+    fn friendly_name(&self) -> &str {
+        "TM_MigratePrecompileCode"
+    }
+
+    fn migrate(&self, _available_weight: Weight) -> Weight {
+        log::info!(target: LOG_TARGET, "migrate");
+        let revert_bytecode = vec![0x60, 0x00, 0x60, 0x00, 0xFD];
+
+        let db_weights = T::DbWeight::get();
+
+        let mut count = 0u64;
+
+        for address in Precompiles::used_addresses() {
+            pallet_evm::Pallet::<T>::create_account(address.into(), revert_bytecode.clone());
+            count += 1;
+        }
+        db_weights.reads_writes(0, count)
```

Comment:
I wasn't sure what weight to return

Reply:
Migration weights are not important until we have multi-block migrations, so this is fine.

Reply:
But looking at the source code of create account I count 2 writes and 1 read, so maybe you can use that?